src_c/image.c

```diff
@@ -64,6 +53,15 @@ opengltosdl(void);
     ((flipped) ? (((char *)data) + (height - row - 1) * width) \
                : (((char *)data) + row * width))
 
+static PyObject *extloadobj = NULL;
+static PyObject *extsaveobj = NULL;
+static PyObject *extverobj = NULL;
+
+/* define docs for undocumented (kinda deprecated) functions */
```

Comment:
I don't think these functions are deprecated, or will be. They're still useful, and used fairly widely.

Can you please add these docs into the .rst file, and have them come in via the doc headers (generated with `python setup.py docs`)?

Reply:
> pygame.image.load_basic is always be available, no matter how pygame is compiled. Unfortunately they only support loading .bmp files (not png, jpeg or others). Most people would use pygame.image.load() instead.

Reply:
Ok, so these functions are useful, then. And I had made a PR a while back discouraging their use in the docs.
Ok, it’s understandable that load_basic() might fall useful so I can make the changes you are asking.
What about load_extended and save_extended

Reply:
There could be an error block at the end which can do all the cleanup in one place. There are decrement functions which do not fail if the variable is NULL. Which means on failure, all of them would be decremented (and then set back to NULL).

Reply:
It’s kind of under the wrong block of comments, but I get it :) 
We do not want to decrement these variables at the end of modinit stuff, because they get use different throughout the code too.

Reply:
Fixed the refcount issue, and handling errors clearly. As to any discussion on the usage of the load_basic and docs stuff, we can continue that outside this review block (it’s  a bit confusing in here)